### ISSUE TITLE
FilePicker does not automatically select the file when connecting to Jupyter

### ISSUE THREAD

**Issue by tolgagoren** [ITISFoundation/osparc-issues#245]
## Long story short

If you drag a file into the workspace it creates a Filepicker instance with one file. This can be directed into a Jupyter Notebook. However the first time you do this it, the Filepicker's file will say 'file: null'. In order for the file to appear, you have to back out, click the Filepicker, select the only file, and then go back into Jupyter.

This is probably hard to follow but I can easily show it interactively in a second.

## Expected behaviour

FilePicker automatically presents the correct file when first used.

## Actual behaviour

FilePicker presents 'null' file by default and the only file must be again re-selected from the workspace to appear.

## Steps to reproduce

Happy to demo upon request. It is 100% reproducible.

Note: your environment was attached but will not be displayed
<!--
## Your environment
```json
[
  {
    "name": "osparc-simcore",
    "version": "3bf510c",
    "url": "https://github.com/ITISFoundation/osparc-simcore/tree/3bf510c/"
  },
  {
    "name": "osparc-simcore UI [clean]",
    "version": "a75a741",
    "url": "https://github.com/ITISFoundation/osparc-simcore/tree/a75a741/services/web/client/"
  },
  {
    "name": "qooxdoo-compiler",
    "version": "1.0.0-beta.20200111-0015",
    "url": "https://github.com/qooxdoo/qooxdoo-compiler"
  },
  {
    "name": "osparc",
    "version": "1.0.0",
    "url": ""
  },
  {
    "name": "qooxdoo framework",
    "version": "6.0.0-beta-20200110-0942",
    "url": "http://qooxdoo.org"
  },
  {
    "name": "qx-iconfont-fontawesome5",
    "version": "0.2.0",
    "url": "https://github.com/ITISFoundation/qx-iconfont-fontawesome5"
  },
  {
    "name": "qx-iconfont-material",
    "version": "0.1.7",
    "url": "https://github.com/ITISFoundation/qx-osparc-theme"
  },
  {
    "name": "qx-osparc-theme",
    "version": "0.5.4",
    "url": "https://github.com/ITISFoundation/qx-osparc-theme"
  },
  {
    "name": "socket.io",
    "version": "2.2.0",
    "url": "https://github.com/socketio/socket.io"
  },
  {
    "name": "jsondiffpatch",
    "version": "0.3.11",
    "url": "https://github.com/benjamine/jsondiffpatch"
  },
  {
    "name": "jsonTreeViewer",
    "version": "0.6.0",
    "url": "https://github.com/summerstyle/jsonTreeViewer"
  },
  {
    "name": "DOMPurify",
    "version": "2.0.0",
    "url": "https://github.com/cure53/DOMPurify"
  },
  {
    "name": "svg.js",
    "version": "2.7.1",
    "url": "https://github.com/svgdotjs/svg.js"
  }
]
```

**Comment by KZzizzle** [comment]
@tolgagoren I believe this is no longer the case - is it still happening for you?
